src/main/java/mate/academy/EventManager.java

```diff
@@ -1,19 +1,40 @@
 package mate.academy;
 
+import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+
 public class EventManager {
-    public void registerListener(EventListener listener) {
+    private final CopyOnWriteArrayList<EventListener> list = new CopyOnWriteArrayList<>();
```

Comment:
would you understand what is that `list` and what is it used for, if you saw it in the code? I highly doubt it
```suggestion
    private final CopyOnWriteArrayList<EventListener> listeners = new CopyOnWriteArrayList<>();
```